recipes/bigdft/build.sh

```diff
@@ -0,0 +1,13 @@
+# Files
+cp $RECIPE_DIR/conda.rc .
+
+# Build
+export JHBUILD_RUN_AS_ROOT="please do it"
+mkdir build
+cd build
+python ../Installer.py -y autogen
+python ../Installer.py -y build -f ../conda.rc
+
+# Activate script
```

Comment:
Ideally, this should be paired with a `deactivate.sh` script that undoes the effect of the activation script, so the env vars set do not permeate into the user session once the env is deactivated.

Reply:
Thanks. Do you know of any projects that have well written `deactivate.sh` scripts that unset environment variables? Because it seems like I'd first need to store all of the original environment variable values somewhere (noting that multiple sessions with different initial conditions could be inside the environment at the same time) for restoring. I can write a script to do all this but I'd like to avoid any pitfalls.

Reply:
You can search Github for projects in the conda-forge organization which use a `deactivate.sh` script https://github.com/search?q=org%3Aconda-forge+deactivate.sh&type=code

Reply:
[This is a good example](https://github.com/conda-forge/intel-compiler-repack-feedstock/tree/main/recipe). The idea is to backup potential values that were set for that env var _before_ you activate. Then on deactivation, restore to the backup _or_, if empty, unset.

You will need to take notes of what the `source`d script is doing, though :)

Reply:
Thank you for the suggestions! Storing in environment variables with the `conda` suffix seems like a good strategy. I've implemented a deactivate script based on this.